lambdas/src/controllers/handlers/profiles/handlers.ts

```diff
@@ -304,3 +304,34 @@ export type ProfileMetadataForSnapshots = {
 type AvatarForSnapshots = {
   snapshots: AvatarSnapshots
 }
+
+function asyncHandler(handler: (req: Request, res: Response, next: NextFunction) => Promise<void>): RequestHandler {
```

Comment:
can we remove this method and put all the code inside `createProfileHandler` instead?